src/ConEmu/VConChild.cpp

```diff
@@ -1685,13 +1685,9 @@ bool CConEmuChild::TrackMouse()
 
 bool CConEmuChild::CheckMouseOverScroll()
 {
-	if (gpSet->isAlwaysShowScrollbar == 0)
+	if (gpSet->isAlwaysShowScrollbar == 0) // 0 -- don't show scrollbar at all
 	{
-		return false; // don't show scrollbar at all
-	}
-	else if (gpSet->isAlwaysShowScrollbar == 1) // 1 -- show always
-	{
-		return true;
+		return false;
```

Comment:
I mean this part. This change would be unclear in history. But it appeared because lines above were changed too.

Perhaps it would be better to ask for changes, but I harried to deploy release with bugfixes before new week.